edgelet/hyper-named-pipe/src/uri.rs

```diff
@@ -149,14 +149,14 @@ mod tests {
 
     #[test]
     fn url_path() {
-        let url = Uri::new("npipe://./pipe/docker_engine", "/containers/json?all=true").unwrap();
+        let url = Uri::new("npipe://./pipe/boo", "/containers/json?all=true").unwrap();
```

Comment:
use "iotedge_moby_engine" instead of "boo" even in tests, right?

Reply:
No this test is validating `Uri`'s ability to parse path and query from a named pipe URI; it doesn't matter what the pipe's name is. In fact it's better to make the pipe's name something silly so that it's easier to see what's important and what isn't.